_toc.yml

```diff
@@ -0,0 +1,18 @@
+format: jb-book
+root: README
+parts:
+  - caption: Getting started
+    chapters:
+    - file: docs/get-started
+    - file: docs/how-to-contribute
+  - caption: AIOps projects
+    chapters:
+    - file: docs/aiops-projects
+    - file: docs/project-document-template
```

Comment:
Can we re-title this here or change the header name on the doc to "Project outline template"?

Reply:
Afaik, the title is coming from the docs. Would you suggest changing the names of the documents here?

Reply:
yes please : )

Reply:
Awesome, will do that :+1: